examples/plots/Hodograph_Inset.py

```diff
@@ -71,9 +72,16 @@
 
 # Create a hodograph
 ax_hod = inset_axes(skew.ax, '40%', '40%', loc=1)
-h = Hodograph(ax_hod, component_range=80.)
+h = Hodograph(ax_hod, component_range=80.0)
 h.add_grid(increment=20)
-h.plot_colormapped(u, v, np.hypot(u, v))
-
+# create array of data to use for colormapping
+c = np.hypot(u, v)
+# Pass in a custom norm and colormap (from mpl.cm)
+norm = mpl.colors.Normalize(vmin=c.min(), vmax=c.max())
+colors = [mpl.cm.YlOrBr(norm(level)) for level in c]
```

Comment:
Just to get a little more aesthetically pleasing colormap here.
```suggestion
colors = [mpl.cm.viridis(norm(level)) for level in c]
```